riscv-sbi.adoc

```diff
@@ -615,7 +615,13 @@ state of the hart at the time of return value verification.
 | sbi_hart_get_status      	|           2 |     0x48534D
 |===
 
-== Experimental SBI Extension Space, Extension IDs 0x0800000 through 0x08FFFFFF
+== Firmware Code Base SBI Extension Space, Extension IDs 0x0800000 through 0x08FFFFF
+
+Firmware code base (e.g. edk2 or uboot) may define its own SBI extension for the firmware
+drivers built on top of it. This SBI extension space is reserved for the firmware code base 
+specific SBI Extension.
+
+== Experimental SBI Extension Space, Extension IDs 0x0900000 through 0x08FFFFFF
```

Comment:
The beginning ID is after the end ID.
I think you mean:

```suggestion
== Experimental SBI Extension Space, Extension IDs 0x0900000 through 0x09FFFFFF
```

Reply:
Oh! yes. you are right! I will change it later. Thanks